fragmentshandler/fragmentshandler/src/main/java/com/microsoft/device/dualscreen/fragmentshandler/FragmentManagerStateHandler.kt

```diff
@@ -74,6 +69,10 @@ class FragmentManagerStateHandler private constructor(app: Application) : Activi
         }
     }
 
+    override fun onActivityStopped(activity: Activity) {
```

Comment:
REMINDER: Hmm. Here we should test it more, because you can have two activities in onResume state, one activity with transparent background over the other activity. In this case, when the last activity will be closed, the first activity will not receive the events because you have only one Job that is already closed.

Reply:
I've implemented a solution. Please let me know if you agree.